### PR TITLE
[BAU-482] docs: add documentation page for asset card

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

Screenshot 2021-12-08 at 15 29 27

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/components/card/src/asset-card/AssetCard.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    href="https://v4-f36-storybook.netlify.app/?path=/story/components-card-assetcard"
+    target="_blank"
+    alignIcon="end"
+    icon={<ExternalLinkIcon />}
+  >
+    Open in Storybook
+  </TextLink>
+</Stack>
+
+<Props of="AssetCard" />
```
Comment (burakukula) [inline packages/components/card/src/asset-card/AssetCard.mdx:138]:
```suggestion

## Props

<Props of="AssetCard" />
```

Comment (burakukula) [reply packages/components/card/src/asset-card/AssetCard.mdx:138]: should props have title to make the table of contents work?

**Review by burakukula — APPROVED** [review]

File: packages/components/card/src/asset-card/AssetCard.mdx (hunk 1 above)
Comment (gui-santos) [reply packages/components/card/src/asset-card/AssetCard.mdx:138]: They have, check line 125

Comment (gui-santos) [reply packages/components/card/src/asset-card/AssetCard.mdx:138]: This way of doing the title for the props list was introduced in this PR
https://github.com/contentful/forma-36/pull/1579

It's the only way to have a title and a button for storybook aligned

**Review by massao — APPROVED** [review]

**Pull request merged**